conserver/main.c

```diff
@@ -781,8 +781,10 @@ DestroyDataStructures(void)
 
 #if USE_IPV6
     /* clean up addrinfo stucts */
-    freeaddrinfo(bindAddr);
-    freeaddrinfo(bindBaseAddr);
+    if ((struct addrinfo *)0 != bindAddr)
```

Comment:
you don't need the casts.  A NULL or 0 in pointer context can be compared against a pointer of any type.  See C89 section 3.2.2.3.

Reply:
Yes, sure.  Just trying my best to match the existing code style.